index.html

```diff
@@ -20,10 +20,240 @@
+              <td>Starting at 25$</td>
+            </tr>
+            <tr>
+              <td>Colour Services</td>
+              <td>Starting at $65</td>
+            </tr>
+            <tr>
+              <td>Updos</td>
+              <td>Starting at $75</td>
+            </tr>
+            <tr>
+              <td>Hair Extensions</td>
+              <td>By Consultation</td>
+            </tr>
+            <tr>
+              <td>Makeup Application</td>
+              <td>Starting at $65</td>
+            </tr>
+            <tr>
+              <td>Facial & Body Waxing</td>
+              <td>Starting at $15</td>
+            </tr>
+            <tr>
+              <td>Skincare, Lash Extensions + Esthetician Services</td>
+              <td>Varies by Location</td>
+            </tr>
+            <tr>
+              <td>Wedding Services</td>
+              <td>Varies by Location</td>
+            </tr>
+          </tbody>
+
+        </table>
+
+
+      </div>
+
+      <img class="hair-salon" src="hair.salon.png" width="890" height="650">
+
+    </div>
+
+    <!--Sub Section 3: OUR TOP RATED PRODUCTS-->
+    <div class="sub-section-3">
+
+      <div class="section-3-title">
+        <p>OUR TOP RATED PRODUCTS</p>
+      </div>
+
+      <img class="hair-products" src="hair-products.png" width="800">
+
+      <section>
+        <ul class="section-3-content">
+          <li>
+            <section>
+              <p>Kérastase continues to develop custom products and treatments to satisfy the demand for extraordinary
+                hair.</p>
+            </section>
+          </li>
+          <li>
+            <section>
+              <p>Award-winning, premium, easy-to-use haircare. Designed for hairstylists & their clients.</p>
+            </section>
+          </li>
+          <li>
+            <section>
+              <p>GO24.7 offers a complete, designer grooming collection with the essentials to help every man create his
+                own iconic style.</p>
+            </section>
+          </li>
+        </ul>
+      </section>
+    </div>
+
+    <!--HTML form-->
+    <div class="html-form">
+      <img class="mail" src="mail.png" width="80">
+
+      <div class="html-form-title">
+        <p>With Love, From Luxe Hair and Salon</p>
+      </div>
+
+      <strong>Save 15% on your first care purchase online.</strong>
+      <p>Join our newsletter, <underline>With Love, From Luxe Hair and Salon</underline>, for salon </p>
+      <p>news and promotions - plus, 15% off just for joining.</p>
+
+
+      <div class="email">
+        <form action="/server-page-handling-form-data" method="get">
+          <ul>
+            <li>
+              <input type="email" id="email" name="email" placeholder="Email" />
+            </li>
+            <li>
+              <button type="submit">Subscribe</button>
+            </li>
+          </ul>
+        </form>
+      </div>
+
+    </div>
+    </html-form>
```

Comment:
On no - this might have been a typo, but not sure what a closing `</html-form>` means